test/smoke/dapi.js

```diff
@@ -3,36 +3,43 @@ const getNetworkConfig = require('../../lib/test/getNetworkConfig');
 
 const networkConfig = getNetworkConfig();
 
-networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
-  const options = {
-    debug: false,
-    verbose: false,
-    errors: false,
-    warnings: false,
-    seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
-  };
-  describe(`DAPI ${nodeName}`, () => {
-    // getBlockHash height:1
-    it('should respond data from chain', async () => {
-      const dapi = new DAPIClient(options, 3000);
-      const blockHash = await dapi.getBlockHash(1);
-      expect(blockHash).to.be.an('string');
-    });
+const fetch = require('node-fetch');
 
-    // getBestBlockHeight
-    it('should respond data from insight', async () => {
+describe('DAPI', () => {
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(`DAPI ${nodeName}`, () => {
```

Comment:
Maybe just node name will be enough?

Reply:
do you mean 'DAPI nodeName'? it doesn't work

Reply:
fixed